docs/_posts/murat-gunay/2021-09-09-bert_token_classifier_ner_btc_en.md

```diff
@@ -17,12 +17,12 @@ use_language_switcher: "Python-Scala-Java"
 
 ## Description
 
-- This model is trained on Broad Twitter Corpus (BTC) data-set, so that it can detect entities in Twitter-based texts successfully.
-- `BertForTokenClassification()` module, which uses the Deep Learning (`torch`) algorithm, is used to train this model.
-- The embeddings `bert_base_cased` is embedded inside the model so, you don't need to use any embeddings in the NLP pipeline.
+- This model is trained on Broad Twitter Corpus (BTC) dataset, so that it can detect entities in Twitter-based texts successfully.
+- It's based on `bert_base_cased` embeddings, which are included the model so, you don't need to use any embeddings components in the NLP pipeline.
```

Comment:
which are included `in the model, so you don't need to use any embeddinsg component in the NLP pipeline`